dockstore/lambda
# upload check url lambda deploy to s3

Addresses partially https://ucsc-cgl.atlassian.net/browse/SEAB-3023
Related PR is https://github.com/dockstore/dockstore-deploy/pull/304
and https://github.com/dockstore/compose_setup/pull/207

To verify:
See steps at https://github.com/dockstore/dockstore-deploy/pull/304 verifying that PR ticket should verify this ticket also.
